alan-turing-institute/scivision
# Add scivision connections workshop models and datasets

Closes https://github.com/alan-turing-institute/DS4S-project-management/issues/69

Adds two new models and two new datasets.

### To test

- Install scivision from this branch
- Check out the new [gallery example notebooks](https://github.com/scivision-gallery/connections-workshop-examples) and see if they work

### TODO

-  [x] Use the contributor bot
